distributed/utils_test.py

```diff
@@ -590,15 +590,13 @@ def cluster(
     nanny=False,
     worker_kwargs=None,
     active_rpc_timeout=10,
-    shutdown_timeout=20,
     scheduler_kwargs=None,
     config=None,
 ):
     worker_kwargs = worker_kwargs or {}
     scheduler_kwargs = scheduler_kwargs or {}
     config = config or {}
 
-    ws = weakref.WeakSet()
```

Comment:
Unused in function (and `workers_by_pid` already held strong references to the Process instances anyway). Plus we do want strong references, so that we can kill them at the end. It's not like garbage-collecting a Process instance means the actual OS process will be cleaned up.